### DISCUSSION TITLE
【HEXO主题】增加站内搜索功能，cf部署的站点单个页面刷新发生错误

### DISCUSSION THREAD

**Discussion by shadowstar2023** [tangly1024/NotionNext#2131] [Ideas]
**HEXO主题可以带站内搜索吗？

还有就是cf部署的站点，post页面刷新后会发生错误**
![image](https://github.com/tangly1024/NotionNext/assets/130375230/f84f9b3c-b35f-4938-bac2-b667710d6465)

**Comment by tangly1024** [comment]
> cf部署的站点单个页面不能刷新，一刷新就发生错误

这个问题我暂时没有遇到，我需要更多信息，例如您的配置和详细的操作步骤。


> hexo 站内搜索

我可以为hexo主题加入algolia搜索功能

**Comment by shadowstar2023** [comment]
### cf部署的站点单个页面不能刷新，一刷新就发生错误
应该是我添加顶级域名记录是用的CNAME，所以才不能回源的吧
